helm/factset-uploader/templates/pod.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  # Using a unique name for the pod, so that we can install it multiple times. This will be run on demand whenever we install
```

Comment:
Does work on cluster restarts? I can't see the `upp-factset-uploader` in the pod lists even with `get pods -a`. 
Also the `di2-factset-uploader` didn't start up.

Reply:
The finished pods seem to vanish at some point. Even after each deploy of the helm chart, the old pod can not be found any more. I consider this a benefit as we won't fill in the history with old pods.

The di2 didn't start as the PVC didn't manage to claim the PV, as the PV had already a claimref. I think this is a k8s bug that will be fixed.